src/tests/multihost/alltests/test_config_merging.py

```diff
@@ -33,6 +33,7 @@ def test_0001_verifypermission(self, multihost):
         content = "[%s]\nuse_fully_quailified_name = False" % section
         snippet_file = "/etc/sssd/conf.d/01_snippet.conf"
         multihost.client[0].put_file_contents(snippet_file, content)
+        multihost.client[0].run_command(f"chmod o-rwx {snippet_file}")
```

Comment:
This is wrong, as it doesn't specify ug mode now.
Use 640

Reply:
If you need a config file with correct access mode, of course (I didn't read test details).